### DISCUSSION TITLE
Auto-purging of Junk folder?

### DISCUSSION THREAD

**Discussion by glennzilla** [afterxleep/Bouncer#69] [Q&A] [answered]
Great app -- effectively moves spam texts to Junk folder (once the regex is properly configured).  Do the spam messages sit in the Junk folder forever (until manually deleted), or are they ever auto-purged?

**Comment by afterxleep** [comment, marked as answer]
Messages are kept in the Junk folder unless you delete them or have explicit auto-delete in settings.

For auto-delete, you can go to Go to Settings > Messages > Keep Messages, but that will delete all messages independently if they are junk or not.
